src/pages/steps/StepsForm.tsx

```diff
@@ -178,43 +199,70 @@ const StepsForm = ({
           visible={!!toast.message}
           onClose={() => toast.clear()}
         />
-        <Steps
-          configurations={configurations}
-          onChange={handleChange}
-          fieldLoading={fieldLoading}
-          onChangeSuccess={handleChangeSuccess}
-          offerId={offerId}
-          mainLanguage={offer?.mainLanguage}
-          scope={scope}
-          form={form}
-        />
+        {needsLocationMigration ? (
+          <>
```

Comment:
In general you shouldn't use fragments in our codebase. Because our ui lib relies on passing margin props to its children. When you use a fragment instead of `Stack` the spacing prop of the parent will not work.

Reply:
Oooh good to know 👍